api/app/models/event.py

```diff
@@ -74,7 +74,7 @@ class UserMessage(Message, table=True):
         foreign_key="persona.uid",
         description="The optional persona ID for user messages",
     )
-    persona: Optional[Persona] = Relationship(back_populates="user_messages")
+    persona: Optional[Persona] = Relationship()
```

Comment:
removed any back_populates that I don't think we'll explicitly use; events doesn't really matter because the next PR redoes them anyway